tests/integration/models/device.spec.coffee

```diff
@@ -978,14 +978,19 @@ describe 'Device Model', ->
 				application1: resin.models.application.create
 					name: 'FooBar'
 					applicationType: 'microservices-starter'
-					deviceType: 'raspberry-pi'
+					deviceType: 'raspberrypi3'
 				application2: resin.models.application.create
 					name: 'BarBaz'
 					applicationType: 'microservices-starter'
-					deviceType: 'raspberry-pi'
+					deviceType: 'raspberrypi3'
+				application3: resin.models.application.create
+					name: 'BazFoo'
+					applicationType: 'microservices-starter'
+					deviceType: 'raspberry-pi2'
```

Comment:
We should update the `describe` above this, which is now wrong (there's now 3 applications)

Reply:
Yea missed that :laughing: